scripts/nsls2-dev-build.sh

```diff
@@ -32,8 +32,8 @@ show_channel_urls: true
 channels:
 - $UPLOAD_CHANNEL
 - lightsource2-tag
-- conda-forge
-- defaults" > $CONDARC_PATH
+- defaults
+- conda-forge" > $CONDARC_PATH
```

Comment:
That's also one of my concerns. But this is a temporary solution after discussing with conda-forge team. We will move back when conda team fixed the bugs.